apprentice-react-o-shop/src/components/Auth/LoginForm.js

```diff
@@ -19,9 +19,13 @@ function LoginForm() {
     const submitLoginForm = (event) => {
         event.preventDefault();
         setloginFormSubmitted(true);
-        localStorage.setItem('user', username)
-        localStorage.setItem('password', password)
-        history.push('/home')
+        const loginCheck = JSON.parse(localStorage.getItem('login'))
+        if (loginCheck.user === username && loginCheck.password === password){
```

Comment:
Is there any instance where the username/password won't match what's in local storage? If not then we can just check for `(loginCheck.user && loginCheck.password)`